learning/basics/task1/AC-input_print.py

```diff
@@ -0,0 +1,18 @@
+x = input("Hello! What is your name? ")
+print("Hello " + x + "!, give me two numbers and i will multiply them.")
```

Comment:
`f"Hello {x}!, give me two numbers and i will multiply them."`
As in task description try to use f-string, simply put `f`` before string as above, then you are able to put every variable simply in curly brackets. It is very popular way to put variables in string on real projects.